addons/account/models/account_report.py

```diff
@@ -347,6 +351,18 @@ class AccountReportLine(models.Model):
         ('code_uniq', 'unique (report_id, code)', "A report line with the same code already exists."),
     ]
 
+    def _compute_json_name(self):
+        """Dummy compute"""
+        for l in self:
+            l.json_name='{}'
+
+    def _inverse_json_name(self):
+        for rec in self:
+            json_name = json.loads(rec.json_name)
+            for key in json_name:
+                # TODO: catch error or check if the language is active
+                rec.with_context(lang=key).name = json_name[key]
```

Comment:
You should probably use `TranslationImporter` to batch the translations

Reply:
@william-andre The TranslationImporter seems to use the xml_id all of the time, which is a bit overkill here.